test-containers/bnd-tools-smoke-test/app/src/test/java/com/vaadin/flow/osgi/smoketest/SmokeIT.java

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Wrong header

Reply:
Yeah.
Not sure whether we even need this header in tests. But it should either be removed or another one is used.

I will update my settings in Eclipse for headers in these projects. 
Thanks.